src/Uno/Prism.Uno/Services/Dialogs/IDialogWindow.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Prism.Services.Dialogs
+{
+    public interface IDialogWindow
```

Comment:
Can we add a default implementation of IDialogWindow like on WPF?

Reply:
I'll add that.